manifests/README.adoc

```diff
@@ -15,7 +15,8 @@ to work with _KubeVirt_ in _OpenShift_.
 `crd-offline-virtual-machine.yml`:: Custom resource definition for
 offline virtual machines.
 
-`crd-virtual-machine-template.yml`:: Custom resource definition for
-templates.
+`crd-template.yml`:: Custom resource definition for templates.
```

Comment:
I think this `crd-template.yml` file should be completely deleted, as there will be no `VirtualMachineTemplate` custom resource definition in _KubeVirt_.

Reply:
Agree but we need it for now till the proper implementation is provided.